### DISCUSSION TITLE
load vector map on leaflet

### DISCUSSION THREAD

**Discussion by yaniv14** [IsraelHikingMap/Site#1986] [Q&A] [answered]
Hi,
@zstadler @HarelM 
Is it possible to load the vector map on leaflet?
I know that the tile map from "https://israelhiking.osm.org.il/Hebrew/Tiles/{z}/{x}/{y}.png" is still working, but I don't think its up to date.

**Comment by zstadler** [comment, marked as answer]
The vector tiles are available only on our website and app.
Currently, the raster tiles are updated once a week. You are welcome to make non-commercial use of them in Leaflet.

**Reply by yaniv14** [reply]
Thanks
